src/plugin.ts

```diff
@@ -352,20 +381,24 @@ class User {
             }
             this.bcrypt.genSalt(10, (err, salt) => {
                 this.bcrypt.hash(request.payload.password, salt, (err, hash) => {
-                    request.payload.password = hash;
-                    request.payload.strategy = 'default';
-                    // registration-verify information
-                    request.payload.uuid = this.uuid.v4();
-                    request.payload.verified = false;
-
-                    // dummy picture
-                    request.payload.picture = {
-                        original: "https://achvr-assets.global.ssl.fastly.net/assets/profile_placeholder_square150-dd15a533084a90a7e8711e90228fcf60.png",
-                        thumbnail: "https://achvr-assets.global.ssl.fastly.net/assets/profile_placeholder_square150-dd15a533084a90a7e8711e90228fcf60.png"
-                    };
 
+                    var newUser = {
+                        password: hash,
+                        strategy: 'default',
+                        uuid: this.uuid.v4(),
+                        verified: false,
+
+                        // TODO: retrieve picture from own database
+                        // dummy picture
+                        picture: {
```

Comment:
The frontend should ship the dummy picture. It should be displayed if property `picture` is not present.